Sources/SwiftInspectorVisitors/Tests/StructVisitorSpec.swift

```diff
@@ -1,413 +0,0 @@
-          }
-
-          it("finds FooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "FooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == nil
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.FooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "FooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.FooFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "FooFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.FooFooStruct.BarFooFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarFooFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.FooFooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooClass") {
-            let matching = self.sut.innerClasses.filter {
-              $0.name == "BarFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == ["Equatable"]
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooClass.BarBarFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarBarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.BarFooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooEnum") {
-            let matching = self.sut.innerEnums.filter {
-              $0.name == "BarFooEnum"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooEnum.BarBarFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarBarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.BarFooEnum"
-            }
-
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level structs") {
```

Comment:
This context was brought into the new file under a context named `with a top-level class declaration`